CONTRIBUTING.md

```diff
@@ -141,6 +181,6 @@ Other content files are organized as follows:
 
 When editing these files, maintain the existing structure and frontmatter fields.
 
-### Questions?
+## ❓Questions
 
-The Maintainer Month website is maintained by the Open Source team at GitHub. You can reach out at <maintainermonth @ github .com>
+The **Maintainer Month** website is maintained by the **Open Source team at GitHub**. You can reach out at <maintainermonth@github.com> 💚
```

Comment:
@karasowles now there is a `CODEOWNERS` file. Do you want folks to reach out to the email for questions and all, or tag the maintainers that are listed in the `CODEOWNERS`?